[FEATURE] Add snippets for All and None (#4)
30-seconds/30-seconds-of-csharp: snippets/None.md

---
title: None

> tags: array,lambda,intermediate

Reviewer: ```suggestion
tags: array,lambda,overload,intermediate
```

---

Returns `True` if the provided predicate function returns `False` for all elements in a collection, `False` otherwise.

> Use `Array.Exists()` to test if all elements in the collection return `False` based on `match`.

Reviewer: Maybe rephrase to:
```suggestion
Use `Array.Exists()` to test if all elements in the collection return `False` based on the predicate function, `match`.
```

> Omit the second argument, `match`, to use the overload that checks if each value is `null` by default.

Reviewer: Maybe rephrase to:
```suggestion
Omit the predicate function, `match`, to use the overload that checks if each value is `null` by default.
```
